Composer/packages/client/src/App.tsx

```diff
@@ -21,19 +22,29 @@ const Logger = () => {
 
 export const App: React.FC = () => {
   const { appLocale } = useRecoilValue(userSettingsState);
-  const { fetchExtensions, fetchFeatureFlags } = useRecoilValue(dispatcherState);
+  const userHasNode = useRecoilValue(userHasNodeInstalledState);
 
+  const { fetchExtensions, fetchFeatureFlags, checkNodeVersion } = useRecoilValue(dispatcherState);
+  const [showNodeModal, setShowNodeModal] = useState(false);
   useEffect(() => {
     loadLocale(appLocale);
   }, [appLocale]);
 
   useEffect(() => {
+    checkNodeVersion();
```

Comment:
Should we care about this check only if we are running on electron?

Reply:
We need to check if node is installed in all environments due to the dependency taken on node to create a new bot under the component model which is present in both electron and non electron environments

Reply:
@pavolum on a web hosted composer, your check basically asks server if it has node installed, how's that useful?

Reply:
@pavolum ping